### PR TITLE
add InfraID to admin API

### LINKED ISSUE #1537: Add infraID into adminAPI
When doing adminget we should be able to see infraID field too.
We need to add InfraID field into adminAPI converters so SRE's could check what value it is in the database

### PR DESCRIPTION
**Pull request by rolandmkunkel**
### Which issue this PR addresses:

Fixes #1537

### What this PR does / why we need it:

Adds the InfraID of the Cluster to the Admin API.

Added a test case to the e2e tests.

### CODE REVIEW

**Review by bennerv — CHANGES_REQUESTED** [review]

File: pkg/api/admin/openshiftcluster_convert.go [hunk 1]
```diff
@@ -56,6 +56,7 @@ func (c *openShiftClusterConverter) ToExternal(oc *api.OpenShiftCluster) interfa
 				IntIP:      oc.Properties.APIServerProfile.IntIP,
 			},
 			StorageSuffix: oc.Properties.StorageSuffix,
+			InfraID:       oc.Properties.InfraID,
```
Comment (m1kola) [inline pkg/api/admin/openshiftcluster_convert.go:59]: I think we also need it in `ToInternal`.

File: test/e2e/adminapi_cluster_get.go [hunk 2]
```diff
@@ -27,5 +27,6 @@ var _ = Describe("[Admin API] Get cluster action", func() {
 		// Note: some fields will have empty values
 		// on successfully provisioned cluster (oc.Properties.Install, for example)
 		Expect(oc.Properties.StorageSuffix).ToNot(BeEmpty())
+		Expect(oc.Properties.InfraID).ToNot(BeEmpty())
```
Comment (m1kola) [inline test/e2e/adminapi_cluster_get.go:30]: I think it should not be empty: all new clusters should have it. Also once we run and upgrade with https://github.com/Azure/ARO-RP/pull/1536 (yet to happen) the old clusters will have it too.

**Review by m1kola — CHANGES_REQUESTED** [review]

File: test/e2e/adminapi_cluster_get.go (hunk 2 above)
Comment (m1kola) [reply test/e2e/adminapi_cluster_get.go:30]: Disregard this. I misread the test. It says “To not be empty”. I read it as “to be empty”.

**Review by m1kola — APPROVED** [review]

**Pull request merged**